### DISCUSSION TITLE
Escopo de um programa

### DISCUSSION THREAD

**Discussion by ThiagoRoberto85** [balta-io/forum#107] [Tira Dúvidas] [answered]
<img width="815" alt="Exemplo-atual" src="https://user-images.githubusercontent.com/90941733/201814939-c66e257b-f0c4-49ef-93e0-d119db982507.png">
<img width="837" alt="Exemplo-curso" src="https://user-images.githubusercontent.com/90941733/201814940-3b89a943-30a2-4365-a16f-5ea0494b208d.png">
Olá Balta tudo bem? Estou iniciando seu curso fundamentos de C# e estou com uma dúvida, eu baixei o .Net 6 mas no curso vc usa se não estou enganado o 3.1, logo o escopo Program.cs é diferente. Como faço pra seguir igual o do curso? É só instalar a versão que vc usa no curso e deletar o .Net 6?? Ou consigo escolher qual versão usar?? Não sei se fui claro rsrs, espero que sim.

Abraços!!

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @ThiagoRoberto85, como vai?

Instala o .NET 5 ou 3.1 para ficar na mesma versão do curso e sempre que for criar o projeto, utiliza a flag `-f`.

```
dotnet new console -o MeuConsoleApp01 -f net3.1
dotnet new console -o MeuConsoleApp01 -f net5.0
```

**Reply by ThiagoRoberto85** [reply]
Obrigado pela resposta Balta. Vou fazer dessa forma. Abraços!!

**Reply by andrebaltieri** [reply]
💜
